src/components/ViewResponses/Responses.js

```diff
@@ -0,0 +1,43 @@
+import { Card, Typography } from "antd";
+
+const Response = (props) => {
+  let { responses, userInfo } = props;
+
+  const { Title } = Typography;
```

Comment:
its a good practice to put this outside the component, we dont want to reallocate memory for that var, in each render